modules/objdetect/src/aruco/predefined_dictionaries.hpp

```diff
@@ -20125,8 +20125,8 @@ static unsigned char DICT_7X7_1000_BYTES[][4][7] =
8},{203,5,137,115,7}},{{62,103,242,244,9},{175,206,171,57,6},{146,244,254,103,12},{105,205,87,63,5}},{{80,65,218,214,3},{44,222,132,152,3},{198,181,184,32,10},{193,146,23,179,4}},{{26,21,3,65,5},{86,74,64,45,0},{168,44,10,133,8},{11,64,37,38,10}},{{100,193,140,116,2},{136,151,182,136,0},{66,227,24,50,6},{1,22,222,145,1}},{{10,114,238,195,5},{103,197,76,13,7},{172,55,116,229,0},{235,3,42,62,6}},{{31,15,157,198,0},{154,139,12,121,11},{6,59,159,15,8},{217,227,13,21,9}},{{169,85,155,198,7},{94,214,100,122,3},{230,61,154,169,5},{197,226,102,183,10}},{{243,41,17,208,13},{21,10,225,219,10},{176,184,137,76,15},{93,184,117,10,8}},{{33,192,212,255,12},{40,165,243,92,3},{63,242,176,56,4},{195,172,250,81,4}},{{224,28,239,91,0},{112,225,164,174,13},{13,175,115,128,7},{183,82,88,112,14}},{{78,35,163,82,0},{147,194,136,137,5},{4,172,92,71,2},{169,17,20,60,9}},{{170,79,4,228,9},{14,7,43,43,10},{146,114,15,37,5},{93,77,78,7,0}},{{225,196,252,196,3},{44,149,52,250,6},{194,51,242,56,7},{101,242,202,147,4}},{{32,142,143,110,8},{24,225,63,40,9},{23,111,23,16,4},{145,79,200,113,8}},{{132,134,119,74,5},{180,97,88,58,5},{165,46,230,18,1},{165,193,168,98,13}},{{158,152,199,85,8},{250,201,145,15,8},{26,174,49,151,9},{31,8,153,53,15}},{{44,89,251,125,12},{250,230,231,28,12},{59,237,249,163,4},{51,142,118,117,15}},{{148,70,164,97,3},{132,157,10,46,4},{200,98,86,34,9},{39,69,11,146,1}},{{130,146,220,194,14},{96,145,93,27,3},{116,51,180,148,1},{205,139,168,144,6}},{{0,77,97,99,1},{52,6,2,44,13},{140,104,107,32,0},{179,68,6,2,12}},{{208,85,39,128,9},{84,79,1,162,6},{144,30,74,160,11},{100,88,15,34,10}},{{160,22,56,82,13},{68,0,237,58,5},{180,161,198,128,5},{165,203,112,2,2}},{{143,101,127,99,9},{183,71,7,127,5},{156,111,234,111,1},{175,238,14,46,13}},{{204,166,195,227,7},{183,208,90,174,3},{236,124,54,83,3},{199,85,160,190,13}},{{203,19,107,199,10},{122,82,13,207,7},{94,61,108,141,3},{239,59,4,165,14}},{{252,90,131,229,3},{222,220,42,142,10},{202,124,21,163,15},{87,21,67,183,11}},{{154,164,79,195,0},{51,73,20,47,3},{12,63,34,85,9},{207,66,137,44,12}},{{189,236,27,211,12},{147,76,245,126,11},{60,189,131,123,13},{215,234,243,44,9}},{{224,32,185,247,12},{25,128,231,158,7},{62,249,208,64,7},{231,158,112,25,8}},{{75,143,53,251,0},{18,35,154,253,15},{13,250,207,29,2},{251,245,156,68,8}},{{184,22,95,99,7},{118,89,110,62,1},{236,111,166,129,13},{135,199,105,166,14}},{{51,220,136,214,9},{76,140,181,105,11},{150,177,19,188,12},{217,106,211,19,2}},{{16,162,247,228,13},{61,201,91,24,6},{178,126,244,80,8},{97,141,169,59,12}},{{200,203,95,245,3},{62,87,158,158,10},{202,255,173,49,3},{87,151,158,167,12}},{{222,37,159,246,11},{159,219,135,187,3},{214,255,154,71,11},{205,222,29,191,9}},{{70,208,112,221,4},{232,36,208,157,6},{43,176,224,182,2},{107,144,178,65,7}},{{50,211,185,116,1},{92,142,190,25,4},{130,233,220,180,12},{41,135,215,19,10}},{{112,117,241,192,4},{113,142,96,184,6},{32,56,250,224,14},{97,208,103,24,14}},{{77,88,219,234,0},{242,228,6,216,11},{5,125,177,171,2},{209,182,2,116,15}}};
```

Comment:
Please do not touch irrelevant files. It creates a lot of merge conflicts and noise in the git history.